### PR TITLE
As of Istio v1 CRDs must be installed before Helm

### PR DESCRIPTION
**Pull request by todaywasawesome**
See this issue: https://github.com/istio/istio/issues/7530

### CODE REVIEW

**Comment by todaywasawesome** [comment]
Just signed the cla.

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 1]
```diff
@@ -85,6 +85,11 @@ to manage the lifecycle of Istio.
     {{< text bash >}}
     $ helm init --service-account tiller
     {{< /text >}}
+    
+1. Install CRDs:
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/helm-install/index.md]: Surround the file path with @@ in order to generate a link to the yaml file in GitHub.

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 2]
```diff
@@ -85,6 +85,11 @@ to manage the lifecycle of Istio.
     {{< text bash >}}
     $ helm init --service-account tiller
     {{< /text >}}
+    
+1. Install CRDs:
```
Comment (frankbu) [inline content/docs/setup/kubernetes/helm-install/index.md]: This step isn't needed. It's already done in step 1 of the instructions, above.
https://deploy-preview-2252--preliminary-istio.netlify.com/docs/setup/kubernetes/helm-install/#installation-steps

**Comment by geeknoid** [comment]
Closing this PR, given Frank's comments. Please reopen if you feel otherwise.

**Comment by todaywasawesome** [comment]
Ah, thanks. It wasn't in the docs when I made the PR, must have been in a staging version or something.

**Pull request closed without merging**